pylint/test/functional/stop_iteration_inside_generator.txt

```diff
@@ -0,0 +1,2 @@
+stop-iteration-return:18:classic_generator:Replace StopIteration raising by a return statement
```

Comment:
Slightly better English:

"Do not raise StopIteration in generator, use return statement instead."